data/chameleoncloud/sites/tacc/clusters/chameleon/nodes/fbe7614f-7236-48da-beab-4793a9addbed.json

```diff
@@ -5,125 +5,72 @@
+    "vendor": "Dell",
+    "version": "2.16.3"
   },
   "chassis": {
-    "manufacturer": "Dell Inc.",
+    "manufacturer": "Dell",
     "name": "PowerEdge R6525",
     "serial": "D77CFF3"
   },
-  "infiniband": true,
+  "gpu": {
+    "gpu": false
+  },
   "main_memory": {
-    "humanized_ram_size": "256 GiB",
-    "ram_size": 256000000000
+    "ram_size": 274877906944,
+    "humanized_ram_size": "256 GiB"
   },
   "monitoring": {
     "wattmeter": false
   },
   "network_adapters": [
     {
-      "device": "NIC.Embedded.1-1",
-      "enabled": false,
-      "interface": "Ethernet",
-      "mac": "b0:7b:25:d4:41:7e",
-      "model": null,
-      "rate": 1000000000,
-      "vendor": "Broadcom Inc. and subsidiaries"
-    },
-    {
-      "device": "NIC.Embedded.2-1",
-      "enabled": false,
-      "interface": "Ethernet",
-      "mac": "b0:7b:25:d4:41:7f",
-      "model": null,
-      "vendor": "Broadcom Inc. and subsidiaries"
-    },
-    {
-      "device": "NIC.Integrated.1-1",
-      "enabled": false,
-      "interface": "Ethernet",
-      "mac": "bc:97:e1:80:11:46",
-      "model": "BRCM 4P 1G BT 5720 OCP NIC",
-      "vendor": "Broadcom Inc. and subsidiaries"
-    },
-    {
-      "device": "NIC.Integrated.1-2",
-      "enabled": false,
-      "interface": "Ethernet",
-      "mac": "bc:97:e1:80:11:47",
-      "model": "BRCM 4P 1G BT 5720 OCP NIC",
-      "vendor": "Broadcom Inc. and subsidiaries"
-    },
-    {
-      "device": "NIC.Integrated.1-3",
-      "enabled": false,
-      "interface": "Ethernet",
-      "mac": "bc:97:e1:80:11:48",
-      "model": "BRCM 4P 1G BT 5720 OCP NIC",
-      "vendor": "Broadcom Inc. and subsidiaries"
-    },
-    {
-      "device": "NIC.Integrated.1-4",
-      "enabled": false,
-      "interface": "Ethernet",
-      "mac": "bc:97:e1:80:11:49",
-      "model": "BRCM 4P 1G BT 5720 OCP NIC",
-      "vendor": "Broadcom Inc. and subsidiaries"
-    },
-    {
-      "device": "NIC.Slot.2-1",
+      "device": "ens2f0",
+      "driver": "qede",
       "enabled": true,
       "interface": "Ethernet",
       "mac": "34:80:0d:ed:4f:56",
-      "model": "QLogic 25GE 2P QL41262 Adapter",
-      "rate": 10000000000,
-      "vendor": "QLogic Corp."
+      "model": "FastLinQ QL41000 Series 10/25/40/50GbE Controller",
+      "rate": 25000000000,
+      "vendor": "QLogic"
     },
     {
-      "device": "NIC.Slot.2-2",
+      "device": "ens2f1",
+      "driver": "qede",
       "enabled": true,
       "interface": "Ethernet",
       "mac": "34:80:0d:ed:4f:57",
-      "model": "QLogic 25GE 2P QL41262 Adapter",
-      "rate": 10000000000,
-      "vendor": "QLogic Corp."
-    },
-    {
-      "device": "InfiniBand.Slot.1-1",
-      "enabled": false,
-      "interface": "InfiniBand",
-      "mac": "b8:ce:f6:50:63:f8",
-      "model": null,
-      "vendor": "Mellanox Technologies"
+      "model": "FastLinQ QL41000 Series 10/25/40/50GbE Controller",
+      "rate": 25000000000,
+      "vendor": "QLogic"
     }
   ],
```

Comment:
I am guessing that ironic-inspector at tacc is configured with:
```
add_ports = active
keep_ports = present
```
which will delete entries for ports which did not get configured.

or at least was configured when the nodes were inventoried.